src/app/server.js

```diff
@@ -192,6 +193,84 @@ app.all("/logoff", endSession, (req, res) => {
   });
 });
 
+
+
+
+
+// ===============
+
+var secureToken ;
+var totalRegistration;
+var tokenDetails = {
+  "client_id": "lms",
+  "client_secret": "94196871-d583-407d-a835-d20122c71fbb",
+  "grant_type": "client_credentials"
+};
+
+var formBody = [];
+for (var property in tokenDetails) {
+var encodedKey = encodeURIComponent(property);
+var encodedValue = encodeURIComponent(tokenDetails[property]);
+formBody.push(encodedKey + "=" + encodedValue);
+}
+formBody = formBody.join("&");
+//  fetch('https://nulp.niua.org/auth/realms/sunbird/protocol/openid-connect/token', {
+// method: 'POST',
+// headers: {
+//   "Content-Type": "application/x-www-form-urlencoded"
+// },
+// body: formBody
+// }).then((response) => response.json())
+// .then((responseData) => {
+//   secureToken =responseData.access_token
+//   console.log("responseData-----------",secureToken); 
+// }).catch(err=>{console.log(err)})
+
+
+  app.get("/newRegistrations", ( req,res, next) => {
+    fetch('https://nulp.niua.org/api/user/v1/search', {
+  method: 'POST',
+  headers: {
+    "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJkTEJ3cG5MdE1SVWRCOTVBdWFCWjhMd0hSR2lTUTBpVCJ9.Q7-3SuUgnZXJuu-j2_kw9r8J82ckSxRR6zxylpgVG5o",
```

Comment:
@paritshivani Is there another way to add 'Auth and user-token' dynamically? 

We need to check because hardcode is not the best solution.